docs/docs/develop/devcontainer.md

```diff
@@ -45,6 +45,10 @@ If you need some demo test-data, run the `setup-test` task. This will import an
 
 If you only need a superuser, run the `superuser` task. It should prompt you for credentials.
 
+#### Run background workers
+
+If you want need to process your queue with background workers, run the `worker` task.
```

Comment:
Im not a native speaker, but in my opinion this sentence would sound better without the `want`. @SchrodingersGat what do you think?

Reply:
Good catch. I missed that after rephrasing it :) Fix incoming.

(not an native speaker either, but I didn't want to write it this way)